pkg/util/httpcache.go

```diff
@@ -9,6 +9,23 @@ import (
 	"k8s.io/klog"
 )
 
+// CleanDefaultHTTPCacheDir cleans the default directory used for HTTP caching
+func CleanDefaultHTTPCacheDir() error {
```

Comment:
Can we write a unit test for this function?

Reply:
done